src/main/resources/application.yml

```diff
@@ -0,0 +1,56 @@
+server:
```

Comment:
What does this file do?

Reply:
This is the configuration file for the environment variables in the project. We'll store our details we need to connect to our database, and our auth keys to validate requests, and other environment variables our project needs